runtime/ts/keymgmt/webcrypto.ts

```diff
@@ -0,0 +1,276 @@
+import {DeviceKey, Key, PrivateKey, PublicKey, RecoveryKey, SessionKey, WrappedKey} from "./keys";
+import {KeyGenerator, KeyStorage} from "./manager";
+import idb, {ObjectStore} from 'idb';
+import {decode, encode} from './base64';
+import rs from 'jsrsasign';
```

Comment:
Wondered whether this would in fact be pulled in for use by browser via webpack, and then presuming it is, whether @shans is ok with our carrying further node module js into browser land in this manner.

Alternatively we discussed in person potentially embedding the public key as a constant (would want comment or link to info on how that constant was generated so we can redo or change if needed in future).

Reply:
Discussed in person, leave it as is for now. Currently WebPack prunes it from ArcsLib.js
until it is wired up to the shell. At that time, we could move it if Shane wants?